.github/workflows/alpha-integration-tests.yaml

```
@@ -104,6 +108,47 @@ jobs:
         ./gradlew uploadResults --args="BroadJadeDev.json $outputDir"
         echo "[INFO] Uploading results SUCCEEDED"
         cd ${GITHUB_WORKSPACE}/${workingDir}
+    - name: "[Cherry-pick to Prod GCR] Import Vault Secrets for Production GCR Service Account"
+      uses: hashicorp/vault-action@v2.1.0
+      with:
+        url: ${{ secrets.VAULT_ADDR }}
+        method: approle
+        roleId: ${{ secrets.PROD_ROLE_ID }}
+        secretId: ${{ secrets.PROD_SECRET_ID }}
+        secrets: |
+          secret/dsde/datarepo/production/gcr-sa key | B64_APPLICATION_CREDENTIALS ;
+    - name: "[Cherry-pick to Prod GCR] Authenticate with Prod GCR SA Credentials"
+      env:
+        GOOGLE_APPLICATION_CREDENTIALS: /tmp/prod-gcr-sa.json
+      run: |
+        # write vault tokens
+        base64 --decode <<< ${B64_APPLICATION_CREDENTIALS} > ${GOOGLE_APPLICATION_CREDENTIALS}
+
+        gcloud auth activate-service-account --key-file ${GOOGLE_APPLICATION_CREDENTIALS}
+    - name: "[API][Cherry-pick to Prod GCR] Perform cherry-pick"
+      run: |
+        dev_image=(${GCR_DEV_URL}${API_REPO}:${{ steps.terrahelmfile.outputs.alpha_version_api }})
+        prod_image=(${GCR_PROD_URL}${API_REPO}:${{ steps.terrahelmfile.outputs.alpha_version_api }})
```

Ensure you're concatenating strings and not creating an array with a single element. Also see comment on trailing slash below.

```suggestion
        dev_image="${GCR_DEV_URL}/${API_REPO}:${{ steps.terrahelmfile.outputs.alpha_version_api }}"
        prod_image="${GCR_PROD_URL}/${API_REPO}:${{ steps.terrahelmfile.outputs.alpha_version_api }}"
```

---

Also nit: Switch `dev_image` to `DEV_IMAGE` and `prod_image` to `PROD_IMAGE` to follow shell variable naming guidelines.